privacy/security/data_protection.xml

```diff
@@ -1,31 +1,25 @@
-<?xml version="1.0" encoding="utf-8" ?>
 <!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
      License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
 <odoo>
-    <data noupdate="0">
```

Comment:
This data should remain as noupdate=0 AFAIK.

Reply:
@Yajo, that's the default already, so not really needed.